package.json

```diff
@@ -42,6 +43,6 @@
   },
   "main": "d2l-colors.js",
   "dependencies": {
-    "@polymer/polymer": "^3.0.0"
+    "@brightspace-ui/core": "^0"
```

Comment:
Do you think we should point to `^0.2`?

Reply:
I think so yeah. Like... the `0.` releases are weird because semver treats them as major versions anyway. Which means if we're revving `core` a lot over the next few days/weeks, we'll want to get to `1.0.0` soon I think. Otherwise we'll be in the colors and typography repos all the time bumping it up.